rest_framework/routers.py

```diff
@@ -339,7 +362,7 @@ def get_urls(self):
 
         if self.include_root_view:
             view = self.get_api_root_view(api_urls=urls)
-            root_url = re_path(r'^$', view, name=self.root_view_name)
```

Comment:
should we consider using a if else condition here by using if else for checking both path and re_path for now? if you have a counter logic I am also open for that.

Reply:
As of now the minimum supported django version is 3.0 https://github.com/encode/django-rest-framework/blob/dc300aa4e01c4cc53c5d74057dc307e31d8ef56d/setup.py#L85

I do not think we need an if-else logic to handle the root of `DefaultRouter`, because we are sure that `path` method exists in django > 3 and `path('', ...)` should match the same than `re_path('^$', ...)`.

Reply:
that's great